toyota_nodsu_pt_generated.dbc

```diff
@@ -180,61 +155,89 @@ BO_ 705 GAS_PEDAL: 8 XXX
  SG_ GAS_PEDAL : 55|8@0+ (0.005,0) [0|1] "" XXX
 
 BO_ 740 STEERING_LKA: 5 XXX
- SG_ LKA_STATE : 31|8@0+ (1,0) [0|255] "" XXX
  SG_ STEER_REQUEST : 0|1@0+ (1,0) [0|1] "" XXX
  SG_ COUNTER : 6|6@0+ (1,0) [0|63] "" XXX
  SG_ SET_ME_1 : 7|1@0+ (1,0) [0|1] "" XXX
  SG_ STEER_TORQUE_CMD : 15|16@0- (1,0) [0|65535] "" XXX
+ SG_ LKA_STATE : 31|8@0+ (1,0) [0|255] "" XXX
  SG_ CHECKSUM : 39|8@0+ (1,0) [0|255] "" XXX
 
-BO_ 742 LEAD_INFO: 8 DSU
- SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" HCU
- SG_ LEAD_REL_SPEED : 23|12@0- (0.025,0) [-100|100] "m/s" HCU
- SG_ LEAD_LONG_DIST : 7|13@0+ (0.05,0) [0|300] "m" HCU
+BO_ 742 LEAD_INFO: 8 XXX
+ SG_ LEAD_LONG_DIST : 7|13@0+ (0.05,0) [0|300] "m" XXX
+ SG_ LEAD_REL_SPEED : 23|12@0- (0.025,0) [-100|100] "m/s" XXX
+ SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX
+
+BO_ 765 NOTHING_2FD: 8 XXX
+ SG_ NEW_SIGNAL_1 : 0|64@1+ (1,0) [0|255] "" XXX
+
+BO_ 767 SDSU: 8 XXX
+ SG_ FD_BUTTON : 7|1@0+ (1,0) [0|1] "" XXX
 
-BO_ 835 ACC_CONTROL: 8 DSU
- SG_ ACCEL_CMD : 7|16@0- (0.001,0) [-20|20] "m/s^2" HCU
+BO_ 835 ACC_CONTROL: 8 XXX
+ SG_ ACCEL_CMD : 7|16@0- (0.001,0) [-20|20] "m/s^2" XXX
  SG_ ALLOW_LONG_PRESS : 17|2@0+ (1,0) [0|2] "" XXX
  SG_ ACC_MALFUNCTION : 18|1@0+ (1,0) [0|0] "" XXX
  SG_ RADAR_DIRTY : 19|1@0+ (1,0) [0|1] "" XXX
  SG_ DISTANCE : 20|1@0+ (1,0) [0|1] "" XXX
  SG_ MINI_CAR : 21|1@0+ (1,0) [0|1] "" XXX
- SG_ ACC_TYPE : 23|2@0+ (1,0) [0|3] "" HCU
- SG_ CANCEL_REQ : 24|1@0+ (1,0) [0|1] "" HCU
+ SG_ ACC_TYPE : 23|2@0+ (1,0) [0|3] "" XXX
+ SG_ CANCEL_REQ : 24|1@0+ (1,0) [0|1] "" XXX
  SG_ ACC_CUT_IN : 25|1@0+ (1,0) [0|1] "" XXX
- SG_ PERMIT_BRAKING : 30|1@0+ (1,0) [0|1] "" HCU
- SG_ RELEASE_STANDSTILL : 31|1@0+ (1,0) [0|1] "" HCU
- SG_ ITS_CONNECT_LEAD : 39|8@0+ (1,0) [0|1] "" Vector__XXX
+ SG_ PERMIT_BRAKING : 30|1@0+ (1,0) [0|1] "" XXX
+ SG_ RELEASE_STANDSTILL : 31|1@0+ (1,0) [0|1] "" XXX
+ SG_ ITS_CONNECT_LEAD : 39|8@0+ (1,0) [0|1] "" XXX
  SG_ ACCEL_CMD_ALT : 47|8@0- (0.05,0) [0|0] "m/s^2" XXX
  SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX
 
-BO_ 836 PRE_COLLISION_2: 8 DSU
+BO_ 836 PRE_COLLISION_2: 8 XXX
```

Comment:
@deanlee is cabana supposed to preserve these ECU notations?

Reply:
yes it should be kept, but it's all ignored now. I'll  submit a pr to fix this